apps/site/src/app/(main)/portal/@applicant/components/timeline/TimelineComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { StatusImage } from "./StatusImage";
+
+interface TimelineComponentProps {
+	text: string;
+	finished: boolean;
+	statusIcon: "Accepted" | "Rejected" | "Pending";
+}
+
+export const TimelineComponent: React.FC<TimelineComponentProps> = ({
+	text,
+	finished,
+	statusIcon,
+}) => {
+	const baseStyles = `flex items-center justify-between mb-4 md:mb-8 border-[2px] md:border-[5px] border-[var(--color-white)]`;
+	const colorStyles = finished
+		? "text-[var(--color-white)] bg-[var(--color-black)]"
+		: "text-[var(--color-black)] bg-[var(--color-white)]";
+
+	return (
+		<div className={`${baseStyles} ${colorStyles}`}>
+			<h2 className="text-xs sm:text-base md:text-4xl mx-4 sm:mx-8 md:mx-16 my-5 md:my-[70px]">
+				{text}
+			</h2>
+			<StatusImage statusIcon={statusIcon} finished={finished} />
+		</div>
```

Comment:
Issue: There should be some particle assets to the sides of the Timeline.

![image](https://github.com/user-attachments/assets/908dbca4-a5f4-4d49-86bb-e4d7d115a637)